app/hooks/use-money-input.ts

```diff
@@ -169,10 +169,18 @@ export function useMoneyInput({
 
       if (!rate) throw new Error('Exchange rate not found');
 
-      newInputValue = toMoney({ value: newValue, currency })
-        .convert(state.input.currency, rate)
-        .toString(getDefaultUnit(state.input.currency));
+      let newInputMoney = toMoney({ value: newValue, currency }).convert(
+        state.input.currency,
+        rate,
+      );
 
+      if (newInputMoney.isZero()) {
+        newInputMoney = Money.createMinAmount(state.input.currency);
```

Comment:
this is needed for cases where you have USD account selected and USD as input currency and then you paste the ln invoice with amount that is less than 1 cent (e.g. 1 satoshi). This will make sure that in this case input value is set to min amount for that currency which is 1 cent